receiver/kafkametricsreceiver/internal/metadata/.nocover

```diff
@@ -0,0 +1 @@
+We test the generated code in the package it is used.
```

Comment:
You don't need to add this, we should simply ignore all generated code in codecov